pkg/model/components/etcdmanager/model.go

```diff
@@ -229,6 +218,8 @@ spec:
     emptyDir: {}
 `
 
+const kopsUtilsImage = "registry.k8s.io/kops/kops-utils-cp:1.28.0-alpha.1"
```

Comment:
Should we move the etcd-manager image here too?

Reply:
Probably ... but maybe we do that separately/later?  (Unless you can think of some reason to do it at the same time...)